diff --git a/node_modules/react-native-icon/src/types/AppIcon.tsx b/node_modules/react-native-icon/src/types/AppIcon.tsx
index 87ba507..7086b5c 100644
--- a/node_modules/react-native-icon/src/types/AppIcon.tsx
+++ b/node_modules/react-native-icon/src/types/AppIcon.tsx
@@ -1,6 +1,6 @@
 /* eslint-disable */
 import React, { forwardRef, memo } from 'react'
-import { Image, ImageStyle, StyleProp, View, ViewStyle } from 'react-native'
+import { Image, ImageResizeMode, ImageStyle, StyleProp, View, ViewStyle } from 'react-native'
 // @ts-ignore
 import ActionsHandLower from '../../../../src/icons/actions_hand_lower.svg'
 // @ts-ignore
@@ -344,6 +344,7 @@ export interface AppIconProps {
   readonly height?: number
   readonly size?: number
   readonly scale?: number
+  readonly resizeMode?: ImageResizeMode
 }
 
 // Icon kind: 'svg' or 'image'
@@ -795,7 +796,7 @@ const getSvgIcon = (
 }
 
 const AppIcon = (
-  { type, testID, nativeID, style, containerStyle, isVisible, tint, stroke, color, width, height, size, scale = 1 }: AppIconProps,
+  { type, testID, nativeID, style, containerStyle, isVisible, tint, stroke, color, width, height, size, scale = 1, resizeMode }: AppIconProps,
   ref: React.Ref<View>
 ) => {
   if (isVisible === false) return null
@@ -827,7 +828,7 @@ const AppIcon = (
       <Image
         source={source}
         style={[{ width: imgWidth, height: imgHeight, tintColor: color || tint }, style as ImageStyle]}
-        resizeMode="contain"
+        resizeMode={resizeMode ?? 'contain'}
       />
     )
   }
